src/queries/getUnavailableTimes.js

```diff
@@ -1,8 +1,8 @@
 const databaseConnection = require("../database/db_connection.js");
 
-const getBusyTimeSlots = () => {
+const getBusyTimeSlots = baseDate => {
   return databaseConnection.query(
-    `SELECT booking_time FROM bookings WHERE booking_date='20191203'`
+    `SELECT booking_time FROM bookings WHERE booking_date='${baseDate}'`
```

Comment:
risks of sql injection here, use the `$1` syntax for psql so like 
`
(`SELECT booking_time FROM bookings WHERE booking_date=$1`,[someValue])
`

Reply:
fixed